db/version_edit.h

```diff
@@ -295,7 +303,7 @@ class VersionEdit {
 
   const char* DecodeNewFile4From(Slice* input);
 
-  typedef std::set<std::pair<int, uint64_t>> DeletedFileSet;
+  typedef std::vector<std::pair<int, uint64_t>> DeletedFileSet;
```

Comment:
shall we use `unordered_set` to be safe? how much more latency will it add?

Reply:
`unordered_set` will add extra `10ms` for inserting 18k delete files comparing to `vector`.

Reply:
then how about adding a dedup? just to be safe.